### PR TITLE
feat: add initContainers support

### PR DESCRIPTION
**Pull request by georgerapeanu**
This pull requests attempt to add support for initContainers. A brief example on how one would look like. This could be useful, for example, for determining a github app token just before renovate executes.

### CODE REVIEW

**Comment by georgerapeanu** [comment]
@viceice could you please re-run the workflows? :)

File: charts/renovate/values.yaml [hunk 1]
```diff
@@ -158,3 +163,4 @@ hostAliases: []
 #   - ip: "your-ip"
 #     hostnames:
 #       - "your-hostname"
+
```
Comment (viceice) [inline charts/renovate/values.yaml:166]:
```suggestion
#       - "your-hostname"
```

Comment (viceice) [reply charts/renovate/values.yaml:166]: can you please revert the newline change?

Comment (georgerapeanu) [reply charts/renovate/values.yaml]: done

**Review by viceice — APPROVED** [review]

File: charts/renovate/values.yaml [hunk 2]
```diff
@@ -11,6 +11,11 @@ cronjob:
   jobRestartPolicy: Never
   jobBackoffLimit: ''
   startingDeadlineSeconds: ''
+  # -- Add init containers to be run before renovate
```
Comment (viceice) [inline charts/renovate/values.yaml:14]:
```suggestion
  # -- Additional initContainers that can be executed before renovate
```

**Pull request merged**